doc/source/whatsnew/v1.0.2.rst

```diff
@@ -30,7 +30,7 @@ Bug fixes
 
 **ExtensionArray**
 
-- Fixed issue where taking the minimum or maximum of a ``StringArray`` or ``Series`` with ``StringDtype`` type would raise. (:issue:`31746`)
+- Fixed issue where taking :meth:`min` or :meth:`max` of a ``StringArray`` or ``Series`` with ``StringDtype`` type would raise. (:issue:`31746`)
```

Comment:
```suggestion
- Fixed issue where taking ``min`` or ``max`` of a ``StringArray`` or ``Series`` with ``StringDtype`` type would raise. (:issue:`31746`)
```

(the min/max methods can't be linked to, because they don't exist as html page. And also if they existed, it would need to be ``` :meth:`pandas.arrays.StringArray.min` ```)